core/admin/mailu/configuration.py

```diff
@@ -143,6 +143,7 @@ def init_app(self, app):
         self.config['SESSION_COOKIE_SAMESITE'] = 'Strict'
         self.config['SESSION_COOKIE_HTTPONLY'] = True
         self.config['PERMANENT_SESSION_LIFETIME'] = timedelta(hours=int(self.config['SESSION_LIFETIME']))
+        self.config['HOSTNAME'] = self.config['HOSTNAMES'].split(',', 1)[0].strip()
```

Comment:
Maybe we should normalize HOSTNAMES instead (to avoid having to strip elsewhere in the code too)

Reply:
Yes, I'll do this. But I like to keep the separate HOSTAME variable in the config, so I don' need to spit everywhere.
But I don't know a way to fix/check the environment before used by docker-compose.
After all the splitting is done in every container in start.py or config.py

Reply:
I am normalizing the HOSTNAMES now.
But we should - as discussed - move this to a python library in start.py